src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.ts

```diff
@@ -31,6 +31,8 @@ export class BuilderNavbarComponent implements OnDestroy {
   learningObject: LearningObject;
   collection: Collection;
   statusDescription: string;
+  // FIXME: This will need to set based on the data coming back once the service is in place
+  revisedVersion = false;
```

Comment:
How do we plan to determine this? Will this component have a function that evaluates this? I think it could be a good idea to go as far as we can at the moment to encapsulate the fixme to a smaller scope.

Reply:
@DonSeannelly I am not quite sure what we are going to use to determine this. I can write a function to evaluate this, but I am not sure what is going to be used to determine if the object being opened is an original or revision.